cloud/filestore/libs/daemon/server/config_initializer.cpp

```diff
@@ -33,6 +33,10 @@ void TConfigInitializerServer::InitAppConfig()
     }
 
     ServerConfig = std::make_shared<NServer::TServerConfig>(serverConfig);
+
+    if (Options->DiagnosticsConfig) {
```

Comment:
why do you need this ? 
DiagnostiConfigs already initialized in the base class by calling `TBootstrapCommon::InitCommonConfigs`